boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -30,6 +30,15 @@ OPERATOR_IMAGE_URI=${IMG}
 OPERATOR_IMAGE_URI_LATEST=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):latest
 OPERATOR_DOCKERFILE ?=build/Dockerfile
 
+OLM_BUNDLE_IMAGE = $(OPERATOR_IMAGE)-bundle
+OLM_CATALOG_IMAGE = $(OPERATOR_IMAGE)-catalog
+OLM_CHANNEL ?= alpha
+OLM_BUNDLE_VERSIONS_REPO ?= gitlab.cee.redhat.com/service/saas-deployment-validation-operator-bundle.git
```

Comment:
Shouldn't it be gitlab.cee.redhat.com/service/saas-operator-versions.git ?

Reply:
+1 https://github.com/app-sre/deployment-validation-operator/blob/master/Makefile#L1

Reply:
Though thinking about it, why not just hardcode it in the script itself, since it's going to be the same for every consumer? That'll save us a parameter/envvar :)

Reply:
Good catch. Yes, I will change it. I would leave as parametrizable, I like the possibility of overriding it, e.g in testing scenarios.

Reply:
Fixed in ea1c9f7